app/controllers/event_session_comments_controller.rb

```diff
@@ -0,0 +1,51 @@
+class EventSessionCommentsController < ApplicationController
+  before_filter :authenticate_user!
+
+  def create
+    @event_session_comment = EventSessionComment.new(comment_params)
+    @event_session = EventSession.find(params[:event_session_comment][:event_session_id].to_i)
+    @event_session_comment.user = current_user
+    if @event_session_comment.save
+      redirect_to event_session_path(@event_session), :notice => "Comment added successfully"
+    else
+        redirect_to event_session_path(@event_session), :alert => "Failed to add comment"
+    end
+  end
+
+  def edit
+    @event_session_comment = EventSessionComment.find(params[:id])
```

Comment:
Potential IDOR vulnerability here. What stops a user from edit comment of another user by `id` ?